docs/recipes/vue.md

```diff
@@ -0,0 +1,71 @@
+# Testing Vue components
+
+## Dependencies
+
+- [Require Extension hooks](https://github.com/jackmellis/require-extension-hooks):
+  - `npm i -D require-extension-hooks require-extension-hooks-vue require-extension-hooks-babel`
+
+- [browser-env](browser-testing.md)
+  - `npm i -D browser-env`
+
+## Setup
+
+The first step is setting up a helper to configure the environment to transpile `.vue` files and run in a browser like environment:
+
+```json
+{
+  "ava": {
+    "babel": "inherit",
+    "require": [
+      "./test/helpers/setup.js"
+    ],
+  },
+}
+```
+
+```js
+// Setup browser environment
+require('browser-env')();
+const hooks = require('require-extension-hooks');
+const Vue = require('vue');
+
+// Setup Vue to remove production tip
+Vue.config.productionTip = false;
+
+// Setup vue files to be processed by `require-extension-hooks-vue`
+hooks('vue').plugin('vue').push();
+// Setup vue and js files to be processed by `require-extension-hooks-babel`
+hooks(['vue', 'js']).plugin('babel').push();
```

Comment:
I'm happy for this recipe to be released. Looking at adding a `ava-setup-vue` package. However due to the flexible nature of Vue files will need to be more customisable so you can select transpilers. Which will take a bit of time. This recipe is good fine for a standard Vue setup but not much more